trajectory_msgs/JointTrajectory path
---
bool started